### ISSUE TITLE
[Bug]:  Geany immediately crashes Xvnc

### ISSUE THREAD

**Issue by rallg** [termux/termux-packages#10856]
### Problem description

Using xfce4. Launch Geany from either Menu or command line, and immediately Xvnc crashes, taking down all of xfce4 and dbus.
Repeatable. If launched from GUI menu, the Geany window is momentarily visible before the crash.

This situation has happened for at least a month. A few weeks ago, xfce4-terminal and qterminal also exhibited the same behavior. But now, xfce4-terminal and qterminal work as expected. This suggests that recently, the underlying problem has been fixed for the terminals but not for Geany.

I note that Geany does work in proot-distro archlinux.

Further information: Launched from command line, geany does not report any error. There is a line about $SESSION_MANAGER, but apparently this is expected when launched from command line, and is not the problem. If I run `gdb run geany` then I get `undocumented errno 2` response. In this case, the GUI does not crash, but geany does not open.

### What steps will reproduce the bug?

Launch Genay, either from command line or GUI Menu. Happens immediately, every time.

### What is the expected behavior?

Geany window should appear, and remain open, and functional.

### System information

termux-info:

```
Termux Variables:
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.117
Packages CPU architecture:
arm
Subscribed repositories:
# sources.list
deb https://termux.librehat.com/apt/termux-main/ stable main
# x11-repo (sources.list.d/x11.list)
deb https://packages.termux.org/apt/termux-x11 x11 main
Updatable packages:
All packages up to date
termux-tools version:
0.184
Android version:
11
Kernel build information:
Linux localhost 4.4.177-22626479 #1 SMP PREEMPT Wed Feb 16 19:15:27 +07 2022 armv8l Android
Device manufacturer:
samsung
Device model:
SM-T510
```

**Comment by rallg** [comment]
Further information: Geany was still not working from the Termux packages, as of yeasterday. But then, I successfully built Geany from its git source code. In meson_options.txt I chose 'false' for all bolleans, and 'disabled' for all features. Prior to that, 'clang' and related tools were upgraded, but I doubt if that was the issue. I suspect that the underlying problem has to do with whether Geany has vte enabled, but I am not enough of a programmer to know.
